include/highlight.h

```diff
@@ -189,9 +189,9 @@
 	 * @param c Symbol to be highlighted.
 	 * @param hl Highlighted Line Buffer.
 	 */
-	INLINE static int highlight_symbol(int c, char **hl)
+	INLINE static int highlight_symbol(char c, char **hl)
```

Comment:
If the parameter `c` is then cast to 'unsigned char', the parameter could also be 'unsigned char', right? but OK, not a problem.

Reply:
I think it's a matter of API design – if a function takes `unsigned char` and I have I `char`, I'll be wondering if it's OK to call it, or if I'm doing something wrong. The cast is an implementation detail, which imo shouldn't be reflected in the API.

Reply:
Yes... you're right... I should have made the buffer (`line[]`) as `unsigned char/uint8_t` in the first place...